litmus/director/cluster-connect-check/run_litmus_test.yml

```diff
@@ -26,7 +26,11 @@ spec:
           - name: ANSIBLE_STDOUT_CALLBACK
             value: default  
           - name: DIRECTOR_CLIENT_NAMESPACE
-            value: maya-system     
+            value: maya-system    
+          - name: PROVIDER
+            value: gcp
+          - name: CLUSTER_NAME
+            value: oep-cluster 
```

Comment:
@amitbhatt818 `CLUSTER_NAME` will be the name with which the self-connect cluster gets connected. `oep-cluster` is the cluster name under which we are connecting an external cluster.